app.rb

```diff
@@ -76,6 +141,7 @@ def create_book
     author = gets.chomp
     book = Book.new(title, author)
     @books.push(book)
+    preseve_book
```

Comment:
- [ ] Hello @Team, fantastic job on incorporating the feature to save items to JSON! 🚀👏 I've observed a slight deviation from the project requirements, though. As per the specifications, data should only be saved upon exiting the app—specifically, when the user chooses to exit. In the current implementation, items are being saved immediately after creation, contrary to the desired behavior. 🤔 Kindly take note of this requirement and adjust the code accordingly to ensure that items are saved to JSON only upon exiting the app. 🔄 Thank you! 😊

![Screenshot from 2023-11-30 11-36-01](https://github.com/yayaismaail/oop-school-library/assets/112261237/11db110c-8c67-4543-b57f-684b7ba8ad17)

Reply:
Hi @zeff96, thank you for your optional suggestions and thorough feedback, and please, we need your assistance to provide us a hint to achieve the requirements.
Accept our deepest thanks.
( yayaismaail and Esmatullah12 )

Reply:
@yayaismaail, the simplest way to achieve this would be to create a method that is called when exiting the app

```sh
def exit_program
  save_book
  save_person
  save_rental
  puts "Thanks for using the application"
end
```

and call this method when option for exiting the app is called. Get the picture?

Reply:
Please @zeff96, may I call you via zoom?

Reply:
please go ahead, I was helping other students but done

Reply:
Thank you @zeff96